hardware_interface/test/test_component_parser.cpp

```diff
@@ -183,27 +183,35 @@ class TestComponentParser : public Test
     </hardware>
     <joint name="joint1">
       <classType>ros2_control_components/MultiInterfaceJoint</classType>
-      <commandInterfaceType>position</commandInterfaceType>
-      <commandInterfaceType>velocity</commandInterfaceType>
-      <commandInterfaceType>effort</commandInterfaceType>
+      <commandInterfaceType name="position">
+        <param name="min_value">-1</param>
+        <param name="max_value">1</param>
```

Comment:
The names should be changed to `min` and `max`. You can just copy-paste this from [the document in the roadmap.](https://github.com/ros-controls/roadmap/blob/master/design_drafts/components_architecture_and_urdf_examples.md#ros2-control-components-architecture-and-urdf-description-examples). 

Please do this for all examples.

```suggestion
        <param name="min">-1</param>
        <param name="max">1</param>
```